### PR TITLE
assertive deprecated from cran

### PR DESCRIPTION
**Pull request by krpaulson**

### CODE REVIEW

File: R/agg_helpers.R [hunk 1]
```diff
@@ -61,7 +61,7 @@ agg_categorical <- function(dt,
                  "* See `missing_dt_severity` argument if it is okay to only make ",
                  "aggregate/scale data that are possible given what is available.\n",
                  paste0(capture.output(missing_dt), collapse = "\n"))
-        assertive::assert_engine(empty_missing_dt, missing_dt,
+        assertive.base::assert_engine(empty_missing_dt, missing_dt,
```
Comment (spencerpease) [inline R/agg_helpers.R:64]: It looks like we need to come up with an alternative to signaling different severity levels before we can fully remove `assertive`.

File: R/agg_scale.R [hunk 2]
```diff
@@ -516,7 +516,7 @@ assert_agg_scale_args <- function(dt,
 
     }
   } else {
-    assertthat::assert_that(assertive::is_null(mapping),
+    assertthat::assert_that(is.null(mapping),
```
Comment (spencerpease) [inline R/agg_scale.R:519]: For later: I don't think `assertthat` buys us anything over `checkmate` and we should also consider switching that over as well

**Review by spencerpease — APPROVED** [review]
Strange that `assertive` is being removed from CRAN but `assertive.base` is still around. Anyways, thanks for making this update!

File: R/agg_helpers.R (hunk 1 above)
Comment (krpaulson) [reply R/agg_helpers.R:64]: Yeah exactly. I got stuck on this but assertive.base seems to work for now.

**Pull request merged**